### DISCUSSION TITLE
select里 label一样怎么处理

### DISCUSSION THREAD

**Discussion by langziyang** [DouyinFE/semi-design#1655] [Q&A]
只是label一样，value不一样
![image](https://github.com/DouyinFE/semi-design/assets/2345387/5932b9c0-087b-4c02-badb-beaa71b3cd42)

**Comment by pointhalo** [comment]
FAQ 有对该行为做说明。但建议最好还是不要出现重复 label，这个在现实线下表单里基本不可能存在，是反用户直觉的，用户根本就无法区分两者。

![image](https://github.com/DouyinFE/semi-design/assets/88709023/d94dbd17-ea1a-4a48-8a6b-b24f06c8248b)
